### PR TITLE
JS -> TS, Next -> Bolt & a General Revitalisation of Scrappy

### PR DESCRIPTION
**Pull request by KhushrajRathod**
- Use TypeScript (v important)
- Use Bolt.js and not express (uses express internally)
- Remove duplicate emojiKeywords

### CODE REVIEW

**Comment by sampoder** [comment]
Wowza, we've got a lot here

**Comment by KhushrajRathod** [comment]
A lot of it is still broken -- kinda waiting for @MatthewStanciu, he seems to have removed the public exports for all the api-utils.ts methods, so I don't know what's going on with that

**Comment by MatthewStanciu** [comment]
what you mean

**Comment by KhushrajRathod** [comment]
You removed the export for 't' so now its a private function I can't import -- I can change that ofc but I'm wondering if you intend to scrap that file completely?

File: src/api/startup.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { WebClient } from '@slack/web-api'
+import { t } from '../lib/api-utils'
```
Comment (KhushrajRathod) [inline src/api/startup.ts:2]: See ^ (gh actions fails here)

**Comment by KhushrajRathod** [comment]
Ah that was a mistake, okay

**Comment by MatthewStanciu** [comment]
wait why was this closed

**Comment by MatthewStanciu** [comment]
oh because it was going to be merged into the now-deleted express-switch branch i see

**Comment by KhushrajRathod** [comment]
ah okay, i'll make a fresh PR to main anyway since its going to be cleaner to do with express

**Comment by cfanoulis** [comment]
@KhushrajRathod are you still doing this? I'd like to pick it up

**Pull request closed without merging**